include/aikido/robot/util.hpp

```diff
@@ -200,6 +200,32 @@ trajectory::TrajectoryPtr planToEndEffectorOffset(
     = VectorFieldPlannerParameters(),
     const CRRTPlannerParameters& crrtParameters = CRRTPlannerParameters());
 
+/// Plan with a desired end-effector twist.
+/// \param[in] space StateSpace for the metaskeleton
+/// \param[in] metaSkeleton Metaskeleton to plan with
+/// \param[in] body Bodynode for the end-effector
+/// \param[in] twistSeq Twist for the end-effector
+/// \param[in] durationSeq Time to plan with the desired twist
+/// \param[in] collisionTestable Collision constraint to check. Self-collision
+/// is checked by default.
+/// \param[in] timelimit Timelimit for the plan to be generated
```

Comment:
How is this different from the docstring for `durationSeq`? Both seem to suggest planning timeout